src/go/rpk/pkg/cli/transform/meta.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+package transform
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newPauseCommand(fs afero.Fs, p *config.Params) *cobra.Command {
```

Comment:
Why do we call those functions as "new"? Do we have an "old" pause?

Reply:
Common parlance for "make me one of these". That is, the effect of this function is to create a "new" instance of a "PauseCommand", which is an abstract type of thing.